inst/workflow/3_reporting/Results.yaml

```diff
@@ -25,4 +25,16 @@ steps:
       lAnalysis: lAnalyzed
       strName: "Analysis_Summary"
       strStudyID: strStudyID
+      dSnapshotDate: dSnapshotDate
+  # Stack latest snapshot with previous snapshots.
+  - output: Reporting_Results_With_Changes
+    name: gsm.reporting::CalculateChange
+    params:
+      dfResults: Results_Summary
+      dfResultsLongitudinal: Reporting_Results_Longitudinal
```

Comment:
After taking another look at this, does `Reporting_Results_Longitudinal` need to be in the spec? Because otherwise how does it recognize this object? But then that becomes a little circular because longitudinal is built off of results? Perhaps this should be spun into a second yaml with priority 2?